site/_posts/2019-10-09-Custom-E2E-Image.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Simple Approaches to Customizing the Kubernetes E2E Tests
+image: /img/vmware-tanzu.svg
```

Comment:
Is this the image that we are using for all our blog posts from now on?

Reply:
Great question; I just copy-pasted the intro part. @jonasrosland do you mind if we go back to just using the Sonobuoy one? That would be my preference.

Reply:
Moved it back to sonobuoy logo unless told otherwise.

Reply:
No need to use the VMware Tanzu logo except when it's being called out, so yeah, change to the Sonobuoy one, or a more descriptive screenshot, or something else :)